Makefile

```diff
@@ -95,15 +103,20 @@ include common/Makefile.common.mk
 
 ##@ Development
 
+OS    = $(shell uname -s | tr '[:upper:]' '[:lower:]')
+ARCH  = $(shell uname -m | sed 's/x86_64/amd64/')
```

Comment:
We already have `LOCAL_OS` and `LOCAL_ARCH`, should we reuse them?

Reply:
Yes, I'll change it. This was leftover pieces of code I didn't clean up